tests/sonic_xcvr/test_cmis.py

```diff
@@ -0,0 +1,25 @@
+from mock import MagicMock
+
+try:
+    from sonic_platform_base.sonic_xcvr.api.public.cmis import CmisApi
+    from sonic_platform_base.sonic_xcvr.mem_maps.public.cmis import CmisMemMap
+    from sonic_platform_base.sonic_xcvr.xcvr_eeprom import XcvrEeprom
+    from sonic_platform_base.sonic_xcvr.codes.public.sff8024 import Sff8024
+except ImportError as e:
+    raise ImportError("%s - required module not found" % e)
```

Comment:
The `try..except` block seems unnecessary. Suggest removing.

Reply:
Done